README.md

```diff
@@ -853,22 +853,25 @@ Some initial set up of your project repo is required. This involves having a lam
 
 6. Add the AWS User Environment variables to your Codeship project. In the Environment tab in Codeship, add your 'AWS_ACCESS_KEY', 'AWS_SECRET_ACCESS_KEY' and 'AWS_DEFAULT_REGION' (usually 'us-east-1'). This is needed in order to authorise the Codeship to execute commands from the aws cli.
 
-  ![Environment variables](https://cloud.githubusercontent.com/assets/5912647/12619073/63294abe-c50e-11e5-9f91-2452078968ef.png)
+  ![Environment variables](https://user-images.githubusercontent.com/5222954/32383573-bdf7ad6e-c08e-11e7-938a-0ab81f78e830.png)
 
 7. Set up Test and Deployment Scripts for your Codeship project
 
   Click the the Test tab in your project settings.
 
-  ![Test script](https://cloud.githubusercontent.com/assets/5912647/12619098/83e071a6-c50e-11e5-8d28-45c8afc19f2e.png)
+  ![Setup Commands](https://user-images.githubusercontent.com/5222954/32383449-63517944-c08e-11e7-860f-aa7740806db3.png)
 
   You should already see the follow default code:
 
   ```bash
-  # By default we use the Node.js version set in your package.json or the latest
-  # version from the 0.10 release
+  # We support all major Node.js versions. Please see our documentation for a full list.
```

Comment:
@kellyjandrews this refers to the Node.js Versions supported by AWS Lambda which is actually only `v4.3.2` and `6.10.3` see: http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html
therefore we deliberately select one of _these_ versions to run on CodeShip to ensure that if tests pass on CodeShip they will _actually_ run on Lambda. 😉

Reply:
Right - I was updating the default text that shows up on the test script config page from Codeship, without changing your original tutorial.